pylintrc

```diff
@@ -19,7 +20,13 @@ disable=
 [OPTIONS]
 good-names=id,log,loader,setUp
 max-args=6
-method-rgx=_?[a-z_][a-z0-9_]{2,40}$
+
+# Naming style matching correct method names
+method-naming-style=snake_case
+
+# Regular expression matching correct method names. Overrides method-naming-
+# style,
+#method-rgx=_?[a-z_][a-z0-9_]{2,40}$
```

Comment:
I think the options here are pretty descriptive, so no need for additional comments. 
```suggestion
method-naming-style=snake_case
```